### PR TITLE
Change-In updates

### PR DESCRIPTION
**Pull request by shiroyasha**

### CODE REVIEW

File: docs/essentials/building-monorepo-projects.md [hunk 1]
```diff
@@ -241,6 +241,24 @@ blocks:
       when: "change_in('../Gemfile.lock')"
 ```
 
+### When any Javascript file changes
+
+```yaml
+blocks:
+  - name: JS tests
+    run:
+      when: "change_in('/assets/**/*.js')
```
Comment (DamjanBecirovic) [inline docs/essentials/building-monorepo-projects.md:250]: The closing quote for the when expression is missing.

File: docs/essentials/building-monorepo-projects.md [hunk 2]
```diff
@@ -241,6 +241,24 @@ blocks:
...
+```
+
+### When any file changes, except docs
+
+```yaml
+blocks:
+  - name: Test Web Server
+    run:
+      when: "change_in('/web-app/', exclude: ['/web-app/docs'])"
```
Comment (DamjanBecirovic) [inline docs/essentials/building-monorepo-projects.md:259]: The `exclude` option needs to be within the configuration map, e.g.
`change_in('/web-app/', {exclude: ['/web-app/docs']})`

File: docs/reference/conditions-reference.md [hunk 3]
```diff
@@ -272,19 +276,31 @@ The supported map parameters are:
...
         triple-dot syntax and same predefined values are available as stated
-        above in the description of <b>branch_range</b> property.  
+        above in the description of <b>branch_range</b> property.
+      </td>
+    </tr>
+    <tr>
+      <td> exclude </td>
+      <td>
+        List of file patterns to exclude from the change-in pattern match.
+        For example, `change_in('/', exclude: ['/docs'])` will evaluate to true
```
Comment (DamjanBecirovic) [inline docs/reference/conditions-reference.md:291]: The `exclude` needs to be within the map, e.g. `change_in('/', {exclude: ['/docs']})`

File: docs/reference/conditions-reference.md [hunk 4]
```diff
@@ -305,6 +321,15 @@ blocks:
       when: "change_in('../Gemfile.lock')"
 ```
 
+### When any file changes, except in the docs directory
+
+```yaml
+blocks:
+  - name: Unit tests
+    run:
+      when: "change_in('/', exclude: ['/docs'])"
```
Comment (DamjanBecirovic) [inline docs/reference/conditions-reference.md:330]: The `exclude` needs to be within the map, e.g. `change_in('/', {exclude: ['/docs']})`

File: docs/reference/pipeline-initialization.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# Pipeline Initialization
+
+Every Semaphore pipeline goes through an initialization process during
+which Semaphore fetches, evaluated, and verifies the Pipeline YAML file.
```
Comment (DamjanBecirovic) [inline docs/reference/pipeline-initialization.md:4]: Shoud be: `evaluates`

**Comment by shiroyasha** [comment]
Updated the docs based on feedback.

**Review by DamjanBecirovic — APPROVED** [review]

**Pull request merged**